src/ChipInput.js

```diff
@@ -253,7 +253,7 @@ class ChipInput extends React.Component {
   }
 
   handleKeyDown = (event) => {
-    this.setState({keyPressed: false})
+    this.setState({keyPressed: false, preventChipCreation: false})
```

Comment:
Why do we need this state and handle the keyDown event? Only checking what the function says in `handleAddChip` should be sufficient.

Reply:
it's all because of the `this.clearInput()` inside `handleKeyUp`
i need some more state to know when to prevent also the `handleKeyUp`